packages/webviz-core/src/players/RandomAccessPlayer.test.js

```diff
@@ -597,6 +597,7 @@ describe("RandomAccessPlayer", () => {
       initialized(): void {
         this._initialized++;
       }
+      isPlaying(): void {}
```

Comment:
I don't think this should be necessary? Where is this coming from?

Reply:
It was part of Robbie's commit here: https://github.robot.car/cruise/webviz/commit/2abbc7e1ec698522d4440dbcf1659f80dc0c2c7b#diff-8560f843168c34bfbe7d4ee8ed01a653